ocs_ci/ocs/node.py

```diff
@@ -1058,3 +1058,25 @@ def node_replacement_verification_steps_ceph_side(old_node_name, new_node_name):
 
     log.info("Verification steps from the ceph side finish successfully")
     return True
+
+
+def generate_node_names_for_vsphere(count, prefix="compute-"):
+    """
+    Generate the node names for vsphere platform
+
+    Args:
+        count (int): Number of node names to generate
+        prefix (str): prefix for node name
```

Comment:
```suggestion
        prefix (str): Prefix for node name
```
Just nitpick to be consistent when you start with capital one line above :)

Reply:
Done